lib/validators/liveValidator.ts

```diff
@@ -93,51 +109,51 @@ export class LiveValidator {
    *
    * @returns {object} CacheBuilder Returns the configured CacheBuilder object.
    */
-  constructor(options?: any) {
+  constructor(optionsRaw?: any) {
 
-    this.options = options === null || options === undefined
+    optionsRaw = optionsRaw === null || optionsRaw === undefined
       ? { }
-      : options
+      : optionsRaw
 
-    if (typeof this.options !== "object") {
+    if (typeof optionsRaw !== "object") {
       throw new Error('options must be of type "object".')
     }
-    if (this.options.swaggerPaths === null || this.options.swaggerPaths === undefined) {
-      this.options.swaggerPaths = []
+    if (optionsRaw.swaggerPaths === null || optionsRaw.swaggerPaths === undefined) {
+      optionsRaw.swaggerPaths = []
     }
-    if (!Array.isArray(this.options.swaggerPaths)) {
-      const paths = typeof this.options.swaggerPaths
+    if (!Array.isArray(optionsRaw.swaggerPaths)) {
+      const paths = typeof optionsRaw.swaggerPaths
       throw new Error(
         `options.swaggerPaths must be of type "array" instead of type "${paths}".`)
     }
-    if (this.options.git === null || this.options.git === undefined) {
-      this.options.git = {
+    if (optionsRaw.git === null || optionsRaw.git === undefined) {
```

Comment:
>(optionsRaw.git === null || optionsRaw.git === undefined [](start = 7, length = 56)

`if(!optionsRaw.git) {}` ?

Reply:
In theory, it's slightly different. I would like to completely eliminate usage of `null`.